composer.json

```diff
@@ -44,6 +44,8 @@
         "mockery/mockery": "1.0.*@dev",
         "fzaninotto/faker": "1.2.*",
         "codeclimate/php-test-reporter": "dev-master",
-        "johnkary/phpunit-speedtrap": "~1.0@dev"
+        "johnkary/phpunit-speedtrap": "~1.0@dev",
+        "fabpot/php-cs-fixer": "^1.10",
+        "phpmd/phpmd": "^2.2"
```

Comment:
Heh that's cool, I also see it has auto-chosen the `^` versioner... what is best?